Create ui-c8y-1020-28-2-disable-edit-duplicate-options-for-global-smartrules-displayed-in-other-views.md (#2456)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-28-2-disable-edit-duplicate-options-for-global-smartrules-displayed-in-other-views.md

---
date: ""

> title: Disable edit and duplicate options for global smart rules displayed outside the global smart rules view

Reviewer: ```suggestion
title: Edit and duplicate options for global smart rules disabled outside the Global smart rules page
```

> title: Disable-edit-duplicate-options-for-global-smartrules-displayed-in-other-views-than-global-smartrules-view [GRAFT][release/cd] (#7066)

Author: The Claude AI suggested to use the following title:
```suggestion
title: disable edit and duplicate options for global smart rules displayed outside the global smart rules view
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-55915
version: 1020.28.2
---

> Disable-edit-duplicate-options-for-global-smartrules-displayed-in-other-views-than-global-smartrules-view [GRAFT][release/cd] (#7066)

Author: The Claude AI suggested to use the following body:
```suggestion
In the UI, global smart rules were shown with edit and duplicate options in other views than the dedicated global smart rules view. With this change, the edit and duplicate options for global smart rules are now disabled when they are displayed outside of the global smart rules view. This ensures that global smart rules can only be edited and duplicated in the dedicated view to avoid confusion and unintended modifications in other contexts.
```

> Before it was possible to edit global smart rule from any context (device, group or application). With this change edit and duplicate action for global smart rule only available from global smart rule view.

Reviewer 2: ```suggestion
Previously, it was possible to edit global smart rules from any context (device, group or application). With this change to edit and duplicate actions for global smart rules is only possible to do in the **Global smart rule** page.
```

Reviewer 3: @MWindel what about small improvement on suggestion, like: 

> Previously, it was possible to edit global smart rules from any context (device, group or application). With this change to use edit and duplicate actions for global smart rules is only possible from **Global smart rule** page.

> Previously, it was possible to edit global smart rules from any context (device, group or application). With this change to use edit and duplicate actions for global smart rules is only possible from **Global smart rule** page.

Reviewer: ```suggestion
Previously, it was possible to edit global smart rules from any context (device, group or application). With this change editing or duplicating global smart rules is only possible from the **Global smart rule** page.
```
